src/java/com/zimbra/graphql/models/outputs/GQLSessionInfo.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra GraphQL Extension
+ * Copyright (C) 2018 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+package com.zimbra.graphql.models.outputs;
+
+import com.zimbra.common.gql.GqlConstants;
+import com.zimbra.soap.account.type.AuthToken;
+
+import io.leangen.graphql.annotations.GraphQLQuery;
+import io.leangen.graphql.annotations.types.GraphQLType;
+
+/**
+ * The GQLSessionInfo class.<br>
+ * Contains session response information.
+ * @see com.zimbra.cs.session.Session
+ *
+ * @author Zimbra API Team
+ * @package com.zimbra.graphql.models.outputs
+ * @copyright Copyright © 2018
+ */
+@GraphQLType(name=GqlConstants.CLASS_SESSION_INFO, description="Session information")
+public class GQLSessionInfo {
+    protected AuthToken authToken;
```

Comment:
why do we need AuthToken in the session info? I am not able to recall any use-case.

Reply:
It is used to return a response token.